explicit_edge/Makefile

```diff
@@ -1,55 +1,20 @@
-SETUP = ocaml setup.ml
+PASS_NAME = explicit_edge
 
-build: setup.ml setup.data
-	$(SETUP) -build $(BUILDFLAGS)
+.PHONY: all build install clean
 
-doc: setup.ml setup.data build
-	$(SETUP) -doc $(DOCFLAGS)
+all: install
 
-test: setup.ml setup.data build
-	$(SETUP) -test $(TESTFLAGS)
+build:
+	bapbuild -pkg 'cbat_value_set' -tag 'warn(A-48-44)' $(PASS_NAME).plugin
```

Comment:
I recommend adding `-use-ocamlfind` here, which seems to be necessary when using an Ocaml version around `4.05`.